### PR TITLE
Create TORQUE.conf [BA-5845]

### PR DESCRIPTION
**Pull request by arteymix**
I'm proposing here the configuration I use for [TORQUE](https://www.adaptivecomputing.com/products/torque/).

### CODE REVIEW

File: cromwell.example.backends/TORQUE.conf [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# This is an example of how you can use TORQUE as a backend for Cromwell
+
```
Comment (cjllanwarne) [inline cromwell.example.backends/TORQUE.conf]: Could you add a comment here along the lines of:
```sh
# This sample configuration is a community contribution and not officially supported by the Cromwell team. Use at your own risk!
```

**Comment by cjllanwarne** [comment]
Thanks - I'll merge this as soon as the one remaining test finishes

**Comment by arteymix** [comment]
Thanks to you (and all the contributors), Cromwell has been an incredible solution for my project.

**Comment by aednichols** [comment]
Glad to hear it!

**Pull request merged**